definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -48,5 +48,21 @@
     "Label": "Reason for refusal",
     "FieldType": "TextArea",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "28/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "LabelOutcomeOfServiceApplication",
+    "Label": "## **Outcome of service application**",
+    "FieldType": "Text",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "28/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "serviceApplicationPaymentType",
+    "Label": "How will payment be made?",
+    "FieldType": "Text",
+    "SecurityClassification": "Public"
   }
```

Comment:
WIP - This would be removed as definition is to be played in DIV-6213. Added here to get test to pass and a PR going to review layout